.pipelines/submodules-pipeline.yaml

```diff
@@ -210,6 +210,7 @@ stages:
               ai_path_var: NPM_AI_PATH
               ai_id: 014c22bd-4107-459e-8475-67909e96edcb
               ai_id_var: NPM_AI_ID
+              os_sku: ltsc2022
         steps:
           - template: containers/container-template-windows.yaml
```

Comment:
this template was delete in this pr, should this be containers/conatiner-template.yaml?

Reply:
removed